qemu/tests/balloon_hotplug.py

```diff
@@ -79,20 +78,17 @@ def enable_balloon_service():
 
         error_context.context("Hotplug balloon device for %d times" % (i+1),
                               logging.info)
-        out = new_dev.hotplug(vm.monitor)
-        if out:
-            err += "\nHotplug monitor output: %s" % out
-        # Pause
-        time.sleep(pause)
-        ver_out = new_dev.verify_hotplug(out, vm.monitor)
-        if not ver_out:
-            err += ("\nDevice is not in qtree %ss after hotplug:\n%s"
```

Comment:
@huangyum can we del "err = """ (line 66 of balloon_hotplug.py) since we don't use it.

Reply:
Yeah, you are right. I will delete it.